CarShowRoom/cars/api/v1/views.py

```diff
@@ -0,0 +1,67 @@
+from django.core.exceptions import ObjectDoesNotExist
+from rest_framework import status
+from rest_framework.mixins import (
+    CreateModelMixin,
+    ListModelMixin,
+    RetrieveModelMixin,
+    UpdateModelMixin,
+)
+from rest_framework.response import Response
+from rest_framework.viewsets import GenericViewSet
+
+from cars.api.serializers import CarBrandSerializer, CarSerializer
+from cars.models import Car, CarBrand
+
+
+class RetrieveUpdateCreateCarBrand(
```

Comment:
Do not call views using HTTP methods. Use model name + viewset. E.g for car model class should be named as: `CarViewSet`